packages/synapse-react-client/src/components/StorybookComponentWrapper.tsx

```diff
@@ -132,10 +131,8 @@ export function StorybookComponentWrapper(props: {
           {storybookContext.globals.showReactQueryDevtools && (
             <ReactQueryDevtools />
           )}
-          <MemoryRouter>
```

Comment:
Removed the `MemoryRouter` from our Storybook component wrapper because React Router no longer allows nesting one router component inside of another.

This may break some stories. In those cases, we can add a wrapper to the individual story.